### ISSUE TITLE
Format Exception when reading double with many digits when queried with other double, in a precise order (weird case)

### ISSUE THREAD

**Issue by Nesos-ita** [mysql-net/MySqlConnector#1052]
i don't know why it happens and which is the minium ammount of stuff to make it happen but i have a table with 5 fields: int (id, primary key), int, double, int, double, i have a query that read the last three fields and the query fail throwing "Format exception".  
here is the content of the three fields: `0.5128259631245008; 0; -25.821199826559273`  
i use this code to read:    
```
MySqlDataAdapter ad = new MySqlDataAdapter(cmd);
dati = new DataSet();
ad.Fill(dati);
```
the query works fine if i remove the last "8" from the first number but it also works fine if i query only the first number instead of all three and (here is the most weird case) it works also if i invert the two double fields in the query so i write the query in this way: "select field5, field4, field3 from table where id=1" instead of 3,4,5  
here is a stacktrace:  
```
   in MySqlConnector.Core.TextRow.GetValueCore(ReadOnlySpan`1 data, ColumnDefinitionPayload columnDefinition)
   in MySqlConnector.Core.Row.GetValue(Int32 ordinal)
   in MySqlConnector.Core.Row.GetValues(Object[] values)
   in MySqlConnector.MySqlDataReader.GetValues(Object[] values)
   in System.Data.ProviderBase.DataReaderContainer.CommonLanguageSubsetDataReader.GetValues(Object[] values)
   in System.Data.ProviderBase.SchemaMapping.LoadDataRow()
   in System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   in System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   in System.Data.Common.DataAdapter.Fill(DataSet dataSet, String srcTable, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   in System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   in System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   in System.Data.Common.DbDataAdapter.Fill(DataSet dataSet)
   (ad.Fill from my code is here)
```
i got this stacktrace from dnspy because visualstudio stops at my code.  
it crashes on the utf something parse to double because "data" is empty (sorry i don't have the code right now, tomorrow i will edit this line)
library version: ~1.3.2.0~ updated to latest version: 1.3.13 and the bug is still there (stack trace is of the old version)
mariadb version: 10.4.7

**Comment by bgrainger** [comment]
Is it at all possible you're invoking `MySqlCommand.Prepare` for your `MySqlDataAdapter.SelectCommand`?

**Comment by Nesos-ita** [comment]
yeah it's a prepared query:  
MySqlCommand cmd = conn.CreateCommand();
cmd.CommandText = query;
cmd.Parameters.Add(p);
cmd.Prepare();

**Comment by bgrainger** [comment]
If it's prepared, then this is most likely a duplicate of #1018. Are you able to test with 1.4.0-beta.4 (which contains the fix)? Despite the "beta" label, it shouldn't be any less stable than 1.3.13; it mostly has connection-string-related changes.

**Comment by Nesos-ita** [comment]
sure, i can test it, do you have a link with the compiled dll that i can download? so that i replace the one in the Debug directory for a quick test?

**Comment by bgrainger** [comment]
Duplicate of #1018

**Comment by bgrainger** [comment]
Fixed in [1.3.14](https://github.com/mysql-net/MySqlConnector/releases/tag/1.3.14).
